src/externals/mct/mct/m_GlobalSegMap.F90

```diff
@@ -1764,22 +1761,25 @@ subroutine active_pes_(GSMap, n_active, pe_list)
   character(len=*),parameter :: myname_=myname//'::active_pes_'
 
   integer :: count, i, n, ngseg, ierr
-  logical :: new
-  integer, dimension(:), allocatable :: temp_list
-  integer, dimension(:), allocatable :: perm
+  integer :: max_activepe, p
```

Comment:
Hasn't been touched in the mpi-serial side.  But in CIME, all the indentation was changed in f4a104b15